### DISCUSSION TITLE
Change default QMessageBox::warning to QMessageBox::Yes

### DISCUSSION THREAD

**Discussion by ghost** [lxqt/lxqt#2309] [Ideas]
In libfm-qt and elsewhere there are Yes/No dialog boxes for confirming changes. One example I use a lot is Shift+Delete to bypass the trash can. On Windows the idiom is "Shift+Delete, Enter", but for Lxqt you need to use the arrow keys or use the mouse.

It's a relatively straightforward change, if anybody thinks this is a good idea.

**Comment by tsujan** [comment]
This has been asked several times. "No" as default is intentional.

Also, as it has been mentioned repeatedly, unlike KDE, LXQt doesn't add options for everything.
